add new value to ServiceConditionEnumeration

haeckerbaer opened the pull request:
Currently there's no appropriate value for Condition (within PtSituationElement.Consequence as part of an SX message) in case the AccessibilityAssessment, for example of an AffectedStopPoint, is updated as a consequence of a situation.

haeckerbaer committed:
add new value to ServiceConditionEnumeration

github-actions[bot] committed:
Lint and update documentation tables

TuThoThai committed:
Correction of the annotation

* Clarification of the annotation
* Adding '+v2.3' to identify that it is a new value

Aurige commented:
in SIRI FM, we have a more detailed description:
<img width="855" height="503" alt="image" src="https://github.com/user-attachments/assets/2bc9f1b6-c7e3-4ede-aae9-08ef8feeb41e" />
This allows to make the difference in the consequences for mobility Impaired (as in this proposal) but also for other disabilities (visual, auditory, etc.).
I'm Ok with this PR, but want to raise the discussion to check if we shouldn't extend it

TuThoThai reviewed (dismissed):
Approved with the discussion on extending the enumeration started in #201

TuThoThai committed:
Merge branch 'v2.3-wip' into sx_cr_condition

TuThoThai committed:
Adding strong link with AccessibilityAssessment

Clarifying that the MobilityImpaired boolean value should reflect the overall summary of the AccessibilityAssessment of the StopPoint, when available.

thbar commented on xsd/siri_model/siri_situationClassifiers.xsd:
```diff
@@ -271,6 +271,11 @@ Rail transport, Roads and road transport
 			</xsd:enumeration>
+			<xsd:enumeration value="mobilityImpairedAccessLimited">
+				<xsd:annotation>
+					<xsd:documentation>Access is limited for mobility impaired passengers. The value should correspond the the overall summary of the AccessibilityAssessment of the StopPoint, when known. +v2.3</xsd:documentation>
```
```suggestion
					<xsd:documentation>Access is limited for mobility impaired passengers. The value should correspond to the overall summary of the AccessibilityAssessment of the StopPoint, when known. +v2.3</xsd:documentation>
```

thbar reviewed (dismissed):
LGTM. I've proposed a typo fix.

TuThoThai committed:
Update xsd/siri_model/siri_situationClassifiers.xsd

typo in the documentation

Co-authored-by: Thibaut Barrère <thibaut.barrere@gmail.com>

SteffenSuhr reviewed (approved):
According to the Cologne Public Transportation Authority (and to me), that's fine.
